media/js/mozorg/internet-health/privacy-day-egg.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, KonamiCode) {
+    'use strict';
+
+    var $document = $(document);
+    var $body = $('body');
+    var $kzp = $('#kzp');
+
+    new KonamiCode(function() {
+        $kzp.fadeIn('fast', function() {
```

Comment:
It took a little bit of refactoring so the opacity, display, and animation all have different classes. I've also had to allow time for the transition to complete before completely hiding it and stopping the animation. I think an accidental side-benefit of the fadeIn was that it imposed a delay before adding the key binding to remove the easter egg. Without some delay your finger is still finishing a keystroke and ends the animation before it even begins. I think a brief delay before calling `endEgg()` solves that but there may be a better way.